bridge-stp.in

```diff
@@ -212,7 +212,7 @@ case "$action" in
             # Add bridge to mstpd and configure.
             echo "Adding/configuring bridge '$bridge' ..."
             "$mstpctl" addbridge "$bridge" || continue
-            if [ -n "$config_cmd" ]; then
+            if [ -x "$config_cmd" ]; then
```

Comment:
`-t ` doesn't work too well either it seems
```
[ -t mstpctl111 ] ; echo $?
0
```

so, I'd say let's not change this, or if we do want to change this, we can remove the check altogether

Seems the file is installed as executable anyway.
```
install-data-hook:
    .......
    install -m 755 utils/mstp_config_bridge $(utilsdest)
```

@dvlemplgk 
are you using this on a OpenWrt system ?
on OpenWrt, you'd need to use  `$(INSTALL_DIR)  utils/mstp_config_bridge `  in the `Package/mstpd/install` rule and use some sed magic

for reference, we're also using this on a OpenWrt [well, me, where I work ; the other `mstpd` guys, I don't know for sure how/what they're using ]
but we can't use the `mstp_config_bridge` because we rely on `swconfig` [from OpenWrt] to configure stuff, and we added some glue code to be able to handle that ;
I've been meaning to package `mstpd` for OpenWrt, but did not find the time to do it properly, and not sure about the current devices this can be ported to